demos/multi_channel/yolo_v3/CMakeLists.txt

```diff
@@ -0,0 +1,85 @@
+# Copyright (C) 2018-2019 Intel Corporation
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#      http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set(TARGET_NAME "multi-channel-yolo-v3-demo")
```

Comment:
please change '-' signs to '_' in application demo. We do this for other demos for unification.